utils/s2n_blob.h

```diff
@@ -47,21 +47,28 @@ extern int s2n_blob_char_to_lower(struct s2n_blob *b);
 extern int s2n_hex_string_to_bytes(const uint8_t *str, struct s2n_blob *blob);
 extern int s2n_blob_slice(const struct s2n_blob *b, struct s2n_blob *slice, uint32_t offset, uint32_t size);
 
-#define s2n_stack_blob(name, requested_size, maximum)			\
-    size_t name ## _requested_size = (requested_size);			\
-    uint8_t name ## _buf[(maximum)] = {0};				\
-    POSIX_ENSURE_LTE(name ## _requested_size, (maximum));			\
-    struct s2n_blob name = {0};						\
+#define s2n_stack_blob(name, requested_size, maximum)       \
+    size_t name ## _requested_size = (requested_size);      \
+    uint8_t name ## _buf[(maximum)] = {0};                  \
+    POSIX_ENSURE_LTE(name ## _requested_size, (maximum));   \
+    struct s2n_blob name = {0};                             \
     POSIX_GUARD(s2n_blob_init(&name, name ## _buf, name ## _requested_size))
 
-#define S2N_BLOB_LABEL(name, str) \
-    static uint8_t name##_data[] = str;   \
+#define s2n_stack_blob_result(name, requested_size, maximum)    \
```

Comment:
I'm not a huge fan of lower-case macros. Since this is a new one we can follow the convention of the safety macros:

```suggestion
#define RESULT_STACK_BLOB(name, requested_size, maximum)        \
```

Reply:
That makes total sense. I prefixed with `S2N` since that seems like the convention for a macro that is not per se a safety-macro(?).

Reply:
Okay, back to no `S2N` prefix :)